README.md

```diff
@@ -178,10 +178,10 @@ nginx   2/2     2            2           20s
 
 |                        | Kubernetes 1.15 | Kubernetes 1.16 | Kubernetes 1.17 | Kubernetes 1.18 | Kubernetes 1.19 | Kubernetes 1.20 | Kubernetes 1.21 |
 |------------------------|-----------------|-----------------|-----------------|-----------------|-----------------|-----------------|-----------------|
-| Karmada v0.6           | ✓               | ✓               | ✓               | ✓               | ✓             | ✓               | -               |
-| Karmada v0.7          | ✓               | ✓               | ✓               | ✓               | ✓             | ✓               | -               |
 | Karmada v0.8           | ✓               | ✓               | ✓               | ✓               | ✓             | ✓               | ✓               |
-| Karmada HEAD (master) | ✓               | ✓               | ✓               | ✓               | ✓             | ✓               | ✓               |
+| Karmada v0.9           | ✓               | ✓               | ✓               | ✓               | ✓             | ✓               | ✓               |
+| Karmada v0.10.1        | ✓               | ✓               | ✓               | ✓               | ✓             | ✓               | ✓               |
```

Comment:
We don't need to list `patch` release here. Since no feature or interface change would introduce to `patch` release.

```suggestion
| Karmada v0.10        | ✓               | ✓               | ✓               | ✓               | ✓             | ✓               | ✓               |
```

Reply:
Ok, done.